### ISSUE TITLE
Show all comments - Show all answers

### ISSUE THREAD

**Issue by Gilbertdelyon** [humhub/humhub#5770]
1/ When a post has too many **comments**,
 only the last ones are displayed  + a link "show all X comments"
2/ When a comment has too many **replies**,
 only the last ones are displayed  + a link "show all Y comments"

Instead, the link for replies should be "show all Y replies", or "show all Y answers".

**Comment by ArchBlood** [comment]
This should be configurable in your `/protected/config/common.php` file;

https://github.com/humhub/humhub/blob/master/protected/humhub/modules/comment/Module.php#L20-L40

See the documentations for how to setup configurations in your common.php file;

https://docs.humhub.org/docs/admin/advanced-configuration/#file-configuration

**Comment by Gilbertdelyon** [comment]
Sorry,
The question was not about the number of comments or replies but about the link wording.
For **comments** you have **"show all X comments"**
and for **replies** you ALSO have **"show all Y comments"**, so, it can be confusing

I would suggest 2 differents wordings:
For **comments** : **"show all X comments"**
and for **replies** : **"show all Y replies"**

**Comment by luke-** [comment]
@Gilbertdelyon FYI: We are definitely going to majorly revise the comment section in one of the next releases.

**Comment by Gilbertdelyon** [comment]
Tnanks luke,
It sounds good.

**Comment by Gilbertdelyon** [comment]
@luke
In case you would be looking for ideas in order to improve the comment section.
Please have a look to [this video.](https://drive.google.com/file/d/1d-OrzFmEYkBhoTbTBGXa2E1v6iUAFr4O/view?usp=sharing) (6 minutes - not high quality but enough for a demo)
It shows the way it works on our site (150 subscribers).
No hack. Only JS and CSS in our custom theme. 
Hope it can help

**Comment by luke-** [comment]
Thank you! We'll consider this!

**Comment by dantefromhell** [comment]
@Gilbertdelyon 
That's very nice & clean interface experience! Are you hosting your theme/ modifications somewhere publicly accessible?

**Comment by Gilbertdelyon** [comment]
@dantefromhell 
Yes, it works well !  Custom theme is mainly altering the native behaviour by way of javascript "[MutationObserver](https://developer.mozilla.org/fr/docs/Web/API/MutationObserver)" (May be you know).
It's powerful, but it's a kind of "patch on a wood leg". HH is already massively using javascript and it should be implemented in the core, but such kind of hack is beyond my abilities  ( I am only a poor selfmade hobbyist coder!)

Our theme includes many other modifications. Some are specific to our site and first I should remove them before publishing.
May be next week.

See also above comment from Luke "FYI: We are definitely going to majorly revise the comment section in one of the next releases."

Notice: Theme in the demo if for HH1.11
In case you are running HH1.10 you can use the older version (see zip file in this issue https://github.com/humhub/humhub/issues/5299#issue-1001200706 ). It works at least up to 1.10.4

**Comment by Gilbertdelyon** [comment]
@dantefromhell 
Video in [this post ](https://github.com/humhub/humhub/issues/5770#issuecomment-1169240602) displays this theme
[ThemeRRN_220702_demo.zip](https://github.com/humhub/humhub/files/9035494/ThemeRRN_220702_demo.zip).
You can test and let me know the result.
Please notice:
- works  in HH1.11.2 not tested in other HH releases.
- "more comment" link is converted to "more replies" button. It works only for EN and FR languages. To add other languages you have to go in js/script.js file, line 113.
- js and css files are heavily commented, but in french only.
- published without any guaranty. I am only  hobbyist coder. Please forgive mistakes if you see some. 
- Also includes some modifications of "full screen" editor display.
- You will see some latencies when page is refreshed. this comes from latency of "mutationObser" in js script (500ms), No other way, but I think it's acceptable.
